graphql/resolvers.go

```diff
@@ -60,6 +63,49 @@ func (r *queryResolver) Task(ctx context.Context, taskID string) (*model.APITask
 	return &apiTask, nil
 }
 
+func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error) {
+	allProjs, err := model.FindAllTrackedProjectRefs()
+	if err != nil {
+		return nil, errors.Wrap(err, "error retrieving projects")
+	}
+
+	groupsMap := make(map[string][]*restModel.UIProjectFields)
```

Comment:
There's no need for this to be a pointer

Reply:
It has to be to satisfy the return type of the query, which is an array of pointers

Reply:
That's already satisfied by your return type. This is an intermediate type

Reply:
i played with a couple other ways but they either resulted in errors or would require me to convert each value in `groupedProjects` to a pointer, which can just be done how it is here. I think it's ok to leave as is.

Reply:
But those are already pointers, so there's no net difference for the projects you return. Removing it here would allow you to remove the unnecessary pointers in the rest of the function

Reply:
if i remove the pointer then i get this compile error later on in the function because it expects each `UIProjectFields` in the `groupedProjects` array to be a pointer

<img width="964" alt="Screen Shot 2020-01-30 at 10 23 37 AM" src="https://user-images.githubusercontent.com/15262143/73463170-cdf13780-434a-11ea-8a2a-1bf36c5dc288.png">

Reply:
if you are adamant about removing the pointer then can you please pull my local branch, try it yourself, and post the working solution here? I am not able to remove pointer without compile errors.

Reply:
Oh I had removed the pointer from the generated model when I initially did this. Disregard this